src/components/AnalysisDisplay/AverageValueDisplay/averageValueDisplay.css

```diff
@@ -0,0 +1,8 @@
+.CircularProgressbar .CircularProgressbar-path {
+  stroke: url(#idCSS);
+}
+
+.gradient {
+  background: rgb(255,255,255);
+  background: linear-gradient(90deg, rgba(255,255,255,1) 0%, rgba(236,216,113,1) 75%);
```

Comment:
n00b question: do we need rgba since opacity is always 1? Do you plan to tweak it? 👀

Reply:
I always use hex, i copied that from github to try and implement a gradient but i think i wont do it in the end so ill delete